### PR TITLE
Adding damages divided by disaster type

### LINKED ISSUE #87: Divide the disaster damages according to the disaster type
Currently all the disaster damages are aggregated. To model disaster damages following the structure of the paper, it is necessary to divide the damage by disaster type: hydrometeorological, Climatological.

### PR DESCRIPTION
**Pull request by CamSalda**
Closes #87

### CODE REVIEW

**Comment by jessegrabowski** [comment]
You also need to make sure there are no notebooks that are still trying to import from `laos_gggi.data`

**Comment by CamSalda** [comment]
> You also need to make sure there are no notebooks that are still trying to import from `laos_gggi.data`

Great catch @jessegrabowski. I checked. There was only one and I fixed it.

**Comment by jessegrabowski** [comment]
I still don't see the files you moved. You can view the current status of the repo for this PR here: https://github.com/jessegrabowski/laos-climate-change/tree/87_divide_damages/laos_gggi

**Comment by CamSalda** [comment]
@jessegrabowski I think you should be able to see it now

**Review by jessegrabowski — APPROVED** [review]
I'm still not sure why this was necessary, but it looks fine now

**Review by jessegrabowski — CHANGES_REQUESTED** [review]
sorry you have to remove untitled.ipynb

File: laos_gggi/data_functions/emdat_processing.py [hunk 1]
```diff
@@ -54,7 +63,7 @@ def load_emdat_data(data_path="data", force_reload=False):
     df_raw_filtered = df_raw.query(
         "Total_Affected >1000 &  Deaths >100 & Start_Year > 1970"
     )
-
+    #
```
Comment (jessegrabowski) [inline laos_gggi/data_functions/emdat_processing.py:66]:
```suggestion

```

File: laos_gggi/data_functions/emdat_processing.py [hunk 2]
```diff
@@ -32,6 +32,15 @@ def load_emdat_data(data_path="data", force_reload=False):
...
 
+    df_raw["disaster_class"] = df_raw["Disaster Type"]
+    df_raw["disaster_class"].loc[df_raw["Disaster Type"].isin(["Storm", "Flood"])] = (
+        "Hydrometereological"
+    )
+
+    df_raw["disaster_class"].loc[
+        df_raw["Disaster Type"].isin(["Wildfire", "Extreme temperature", "Drought"])
+    ] = "Climatological"
+
```
Comment (jessegrabowski) [inline laos_gggi/data_functions/emdat_processing.py:43]:
```python
disaster_class_dict = {'Storm': 'Hydrometereological', 'Flood': 'Hydrometereological', 'Wildfire': 'Climatological', 'Extreme temperature':'Climatological', 'Drought':'Climatological'}

df_raw['disaster_class'] = df_raw['Disaster Type'].map(disaster_class_dict.get)
```

**Review by jessegrabowski — CHANGES_REQUESTED** [review]
Looked more closely, here's a more efficient way to do what you're doing

File: laos_gggi/data_functions/emdat_processing.py (hunk 2 above)
Comment (CamSalda) [reply laos_gggi/data_functions/emdat_processing.py]: Oh amazing! Actually I was searching for something like this. Changed.

**Review by jessegrabowski — APPROVED** [review]

**Pull request merged**